#request
float32[] joint_states
time[] mov_time
string int_type
---
#response
bool status